README.md

```diff
@@ -187,6 +289,8 @@ IPFS Core is divided into separate subsystems, each of them exist in their own r
 
 IPFS Core is the entry point module for IPFS. It exposes an interface defined on [IPFS Specs.](https://github.com/ipfs/specs/blob/ipfs/api/api/core/README.md)
 
+This particular implementation also provides some extensions to assist with usage in a browser, such as the `Buffer` type, required as input for some API methods, being available as `ipfs.Buffer` on every `IPFS` object.
```

Comment:
Let's make it more of a disclaimer

```
#### Need the Buffer class?

IPFS exposes the Buffer class in every ipfs instance, so that you can create buffers and add them to IPFS just like if you were using it in Node.js.

You can get it at `ipfs.Buffer`
```